src/apispec/ext/marshmallow/schema_resolver.py

```diff
@@ -0,0 +1,218 @@
+from .common import resolve_schema_instance
+
+
+class SchemaResolver:
+    """Resolve Marshmallow Schemas in OpenAPI components and translate to OpenAPI
```

Comment:
Lowercase marshmallow

```suggestion
    """Resolve marshmallow Schemas in OpenAPI components and translate to OpenAPI
```